esmvaltool/diag_scripts/sea_surface_salinity/compare_salinity.py

```diff
@@ -125,7 +125,7 @@ def create_radar_plot(self, data_info, data, reference, reference_alias,
                        letters,
                        color='grey',
                        size=8,
-                       rotation=angles[:-1])
+                       rotation=45)
```

Comment:
@sloosvel could you maybe pls add an inline comment here why rotate by 45degs - I always like comments explaining hardcoded values

Reply:
I am not sure I have a comment for that? It's just a value for the label for the plots to look good, same for the the size number.

Reply:
wait - isn't that applying a rotation of the whole plot by 45degs? Maybe I am overthinking this if it's not, if it is - why is it rotated by 45deg? That's triggering my OCD :laughing:

Reply:
No, it's just the position of the labels in the plot. There is not much to add I added 45 because it's a nice angle, nothing more.

Reply:
Haha ok, no worries then 😄